lib/rucio/db/sqla/migrate_repo/versions/52153819589c_add_rse_id_to_replicas_table.py

```diff
@@ -0,0 +1,47 @@
+# Copyright 2013-2021 CERN for the benefit of the ATLAS collaboration.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# Authors:
+# - Joaquin Bogado <joaquinbogado@gmail.com>, 2021
+
+''' add_rse_id_to_replicas_table '''
+
+
+from alembic import context
+from alembic.op import create_foreign_key, create_index, drop_constraint, drop_index
+
+
+# Alembic revision identifiers
+revision = '52153819589c'
+down_revision = '30fa38b6434e'
+
+
+def upgrade():
+    '''
+    Upgrade the database to this revision
+    '''
+
+    if context.get_context().dialect.name in ['oracle', 'mysql', 'postgresql']:
+        create_foreign_key('REPLICAS_RSE_ID_FK', 'replicas', 'rses', ['rse_id'], ['id'])
```

Comment:
I suppose you do the FK changes for postgres? If so, we should probably isolate it to that.